qiskit/pulse/schedule.py

```diff
@@ -1300,7 +1300,7 @@ def exclude(
     ):
         """Return a new ``ScheduleBlock`` with only the instructions from this ``ScheduleBlock``
         *failing* at least one of the provided filters.
-        This method is the complement of py:meth:`~self.filter`, so that::
+        This method is the complement of :py:meth:`~self.filter`, so that::
```

Comment:
@Eric-Arellano this seems to not create the link to the method `filter` in the the object. Is that intentional?

Reply:
No, that's not intentional. I suppose the issue is `self.`:

```suggestion
        This method is the complement of :py:meth:`~Schedule.filter`, so that::
```

Reply:
Oh, that makes sense! 
Changing it 👍

Reply:
Committed! 
(there was another one missing in `ScheduleBlock`, check commits: https://github.com/Qiskit/qiskit/pull/12604/commits/8556851ad684ccfd86311eef115d5631ed84d1b4 and https://github.com/Qiskit/qiskit/pull/12604/commits/7719f198117f5c0334d3bd0d31eb9ec2fdd7c533)

And the Azure workflow artifacts build, looks and links nicely:

<img width="1430" alt="Screenshot 2024-07-01 at 19 38 08" src="https://github.com/Qiskit/qiskit/assets/109400222/90ad62a9-cfb1-4e0f-b21b-e46c006264e4">